pages/wiki/docserver.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Local containerized documentation server
```

Comment:
More than a "documentation server" this is the asteroidos.org website

Reply:
Also I think we don't have to insist too much on this being containerized and just consider it as the default method to bring up the website.

So maybe "Local bringup of asteroidos.org" or something like this ?

Reply:
Maybe "local containerized asteroidos.org"?  I thought it was important to note that it's containerized so it doesn't, say, require a user to install and configure Apache. Maybe that's too much to try to convey in just the title?

Reply:
I personally don't think we should stress "containerized" in the title because most of the users don't really care whether it's containerized or not, they just want to bring up the website for local development.

I think we should just document the container way as the only way to bring up the website because it bboth simplifies work on the website and also keeps npm's pile of trash under some amount of control.

Reply:
OK, I'm going to go with "Local version of asteroidos.org" if that's OK with you.

Reply:
yep